website/api/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: /getting-started
+---
+
+# Getting Started
+
+To harness the power of ZK, you'll need to:
+
+1. [Write an application for the RISC Zero zkVM][zkvm-quickstart].
+2. [Generate proofs for your zkVM application][bonsai-quickstart].
+3. [Integrate your proofs into on-chain applications][foundry-template].
```

Comment:
interesting option to embed content from one page in another! i've revised to avoid the redundancy, but mb we want to take advantage of that embed option...